src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -616,11 +616,12 @@ export class GuidedTourService {
          * to prevent ExpressionChangedAfterItHasBeenCheckedError
          */
         setTimeout(() => {
-            this.currentTour = cloneDeep(guidedTour);
```

Comment:
`this.currentTour` is checked when resizing the browser. The bug here was the it was set even before we know if the tour should be started automatically or not.